plugins/mmcapture/smb_sessions.h

```diff
@@ -0,0 +1,38 @@
+/* smb_sessions.h
+ *
+ * This header contains the definition of SMB sessions structures
+ * as well as prototypes for smb_sessions.c
+ *
+ * File begun on 2018-12-5
+ *
+ * Created by:
+ *  - François Bernard (francois.bernard@isen.yncrea.fr)
+ *  - Théo Bertin (theo.bertin@isen.yncrea.fr)
+ *  - Tianyu Geng (tianyu.geng@isen.yncrea.fr)
+ *
+ * This file is part of rsyslog.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *       -or-
+ *       see COPYING.ASL20 in the source distribution
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
```suggested
#include "config.h"
```
This include must be the first include in all source files. This is an autoconf requirement. I guess this also triggers the `strndup()` error.